src/main/java/io/vertx/ext/mongo/BulkWriteOptions.java

```diff
@@ -49,7 +51,7 @@ public BulkWriteOptions(boolean ordered) {
 
   /**
    * Copy constructor
-   * 
+   *
```

Comment:
Please configure your IDE to restrain formatting to changed methods.

Reply:
I will. Still purging an unnecessary space is probably fine.